### PR TITLE
Stored XSS in Maxsite-CMS

### PR DESCRIPTION
**Pull request by Tibinsunny**
### ✍️ **Description**
Maxsite-CMS is a PHP base Blogging CMS which allows users to deploy dynamic websites fast and easy. The site is vulnerable to Stored XSS.

### 🕵️‍♂️ Proof of Concept
1:Download the Source Code from [https://github.com/maxsite/cms](https://github.com/maxsite/cms) 
2:Extract the file into an apache server. This documentation will be helpful https://max-3000.com/doc/install
3:Once all the setup is done we are ready to go !!
4:Navigate to localhost/huntr/cms/admin/options . It is a page which allows admin to edit layout and title.
5:The Title is the Vulnerable area. Inject the payload in the title area input box .The payload I used is ```test"></title><script>alert(1)</script>```
4:Click submit button and XSS will be triaged 

hunrtbug1

### 💥 Impact
The admin can add a Co-admin as well as a Sub-admin. So any malicious admin or user[ Sub-admin] can make use of the vulnerability to target other admin/admins and gain access to the victims account.

### ✅ Checklist 
- [x] Created and populated the README.md and vulnerability.json files  
- [x] Provided the repository URL and any applicable permalinks
- [x] Defined all the applicable weaknesses (CWEs)
- [x] Proposed the CVSS vector items i.e. User Interaction, Attack Complexity
- [x] Checked that the vulnerability affects the latest version of the package released
- [x] Checked that a fix does not currently exist that remediates this vulnerability
- [x] Complied with all applicable laws

### CODE REVIEW

**Comment by JamieSlome** [comment]
@Mik317 @mufeedvh @bbeale - would be great to get your opinions on this!

Do we agree with the CVSS of `6.1` - seems like it could be lower?

Can you also please re-base with the `418sec:staging` branch so that the CI checks pass?

**Comment by Tibinsunny** [comment]
Hey, 
@JamieSlome I am kinda busy this week+My laptop is messed up. Really sorry about that. Is it okay if I make the necessary changes after next week? 

Regards

**Comment by JamieSlome** [comment]
@Tibinsunny - sure! 🍰

File: bounties/other/maxsite/vulnerability.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      "PR": "N",
+      "UI": "R",
+      "S": "U",
+      "C": "L",
+      "I": "L",
+      "A": "N",
+      "E": "",
+      "RL": "",
+      "RC": "",
+      "Score": "6.1"
```
Comment (Mik317) [inline bounties/other/maxsite/vulnerability.json:34]: Too high in my opinion since `admin` permissions are required

**Review by Mik317 — APPROVED** [review]
LGTM :smile: :cake:
The fact `admin` only can change the title and therefore lead to `XSS` is a bad point, also because in this case `admin` has control all over the other users, but in any case an attacker could make the website no more ready to be used changing the CSS ... that however had a really marginal impact. Are you able to show different `attack vectors` to define a better impact? :smile:

Cheers,
Mik

**Comment by JamieSlome** [comment]
@Tibinsunny - I have just realised that the disclosure directories were not correct.

They were missing the `/1/` index in front of the `vulnerability.json` and `README.md`.

Feel free to re-submit another PR with the correct files and we will merge it in!

**Pull request merged**